src/hotspot/share/runtime/sharedRuntime.cpp

```diff
@@ -2867,8 +2867,8 @@ void CompiledEntrySignature::compute_calling_conventions(bool init) {
             if (bt == T_OBJECT) {
               // Nullable inline type argument, insert InlineTypeNode::NullMarker field right after T_METADATA delimiter
               // Set the sort_offset so that the field is detected as null marker by nmethod::print_nmethod_labels.
```

Comment:
This comment needs to be updated. It still mentions the `sort_offset`.

Reply:
I think we can do without the second line. The first says we include a null marker, and the second gives true to the null_marker parameter. Pretty straightforward.